.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -76,6 +76,52 @@ Push access to the upstream repository is required in order to publish the new t
   - [ ] `git push ${UPSTREAM_REMOTE} :release-${RELEASE_VER}`
   - [ ] `git branch -d release-${RELEASE_VER}`
 
+- Fedora packaging:
+  - [ ] update the `rust-coreos-installer` spec file in [Fedora](https://src.fedoraproject.org/rpms/rust-coreos-installer)
+    - bump the `Version`
+    - switch the `Release` back to `1%{?dist}`
+    - remove any patches obsoleted by the new release
+    - update changelog
+  - [ ] run `spectool -g -S rust-coreos-installer.spec`
+  - [ ] run `kinit your_fas_account@FEDORAPROJECT.ORG`
+  - [ ] run `fedpkg new-sources <crate-name> <vendor-tarball-name> <coreos-installer-dracut-tarball-name>`
+  - [ ] PR the changes in [Fedora](https://src.fedoraproject.org/rpms/rust-coreos-installer)
+  - [ ] once the PR merges to rawhide, merge rawhide into the other relevant branches (e.g. f35) then push those, for example:
+    ```bash
+    git checkout rawhide
+    git pull --ff-only
+    git checkout f35
+    git merge --ff-only rawhide
```

Comment:
Note `--ff-only` may not be possible if some automated commit happened on a released branch which causes it to diverge from rawhide.

Reply:
Is there a particular case you're concerned about?  In my experience, automation usually only commits to Rawhide.

(Even if we occasionally have to resolve things by hand, I'd rather we use `--ff-only` by default.  We've had multiple cases where we accidentally create merge commits and tangle the branch history.)

Reply:
Yeah, it's pretty rare, but I've seen it happen on !rawhide. I mostly recall because I remember feeling annoyed about it messing up the graph. :)

SGTM re. defaulting to it.